### DISCUSSION TITLE
Ender Dragon health

### DISCUSSION THREAD

**Discussion by CodingPanda0623** [Stardust-Labs-MC/StardustMC#42] [Ideas]
Now that the end fight is over, can we change the Ender Dragon's health back to vanilla? It takes quite a while just to kill one dragon now.

**Comment by aisukuma** [comment]
This has been changed and will take effect in the upcoming restart. However, the health is not the vanilla's amount, but added an extra 100 HP, so it should not take *too* long to kill.
